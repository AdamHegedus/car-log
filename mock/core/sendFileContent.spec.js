describe('sendFileContent', () => {
  it('should return true', () => {
    expect.assertions(1);
    expect(true).toMatchInlineSnapshot('true');
  });
});
