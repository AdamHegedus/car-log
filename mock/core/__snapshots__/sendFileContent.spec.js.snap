// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sendFileContent should return true: true 1`] = `true`;
